File: 3-Leccion-Condicionales-Ciclos/A.Elevando-al-cuadrado-UAM.cpp
#include <iostream>
using namespace std;

int main() {
  int N;
  cin >> N;

  int pasos = 0;
  while (N < 30000) {
    pasos++;
    N *= N;
  }

  cout << N << " " << pasos << endl;

  return 0;
}

File: 3-Leccion-Condicionales-Ciclos/A.Elevando-al-cuadrado-UAM.java
import java.util.Scanner; 

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    int N = scanner.nextInt();
    int pasos = 0;

    while (N < 30000) {
      pasos++;
      N *= N;
    }

    System.out.println(N + " " + pasos);    
  }
}